nncf/openvino/quantization/helpers.py

```diff
@@ -0,0 +1,106 @@
+from nncf.openvino.engine import OVEngine
+from nncf.quantization.params import TargetDevice
+from nncf.common.quantization.structs import QuantizationPreset
+
+
+def _convert_openvino_model_to_compressed_model(model: ov.Model,
+                                                target_device: str) -> pot.graph.nx_model.CompressedModel:
+    """
+    Serializes the provided OpenVINO model and loads the model in the POT representation.
+
+    :param model: The OpenVINO model.
+    :param target_device: The target device.
+    :return: The POT representation of the provided model.
+    """
+    with tempfile.TemporaryDirectory(dir=tempfile.gettempdir()) as tmp_dir:
+        xml_path = str(Path(tmp_dir).joinpath('model.xml'))
+        bin_path = str(Path(tmp_dir).joinpath('model.bin'))
+        ov.serialize(model, xml_path, bin_path)
+        model_config = {
+            'model_name': 'model',
+            'model': xml_path,
+            'weights': bin_path,
+        }
+        pot_model = pot.load_model(model_config, target_device)
+
+    return pot_model
+
+
+def _convert_compressed_model_to_openvino_model(model: pot.graph.nx_model.CompressedModel) -> ov.Model:
+    """
+    Saves the provided POT compressed model and loads it as `openvino.runtime.Model` object.
+
+    :param model: The POT compressed model.
+    :return: The `openvino.runtime.Model`  object which represents the provided model.
+    """
+    with tempfile.TemporaryDirectory(dir=tempfile.gettempdir()) as tmp_dir:
+        paths = pot.save_model(model, save_path=tmp_dir, model_name='model')
+        xml_path = paths[0]['model']
+        bin_path = paths[0]['weights']
+        ie = ov.Core()
+        ov_model = ie.read_model(xml_path, bin_path)
+    return ov_model
+
+
+def quantize_impl(model: ov.Model,
+                  calibration_dataset: DataLoader,
+                  preset: QuantizationPreset,
+                  target_device: TargetDevice,
+                  subset_size: int,
+                  fast_error_correction: bool,
+                  model_type: Optional[str] = None) -> ov.Model:
+    """
+    Implementation of the `quantize()` method for the OpenVINO backend.
+    """
+    pot_model = _convert_openvino_model_to_compressed_model(model, target_device)
+
+    engine_config = {
+        'device': 'CPU',
+        'stat_requests_number': 2,
+        'eval_requests_number': 2,
+    }
+
+    algorithms = [
+        {
+            'name': 'DefaultQuantization',
+            'params': {
+                'target_device': target_device.value,
+                'preset': preset.value,
+                'stat_subset_size': subset_size,
+                'use_fast_bias': fast_error_correction,
+                'model_type': model_type,
+            }
+        }
+    ]
+
+    pot_dataloader = POTDataLoader(calibration_dataset, calibration_dataset.transform_fn)
+    engine = OVEngine(engine_config, pot_dataloader, pot_dataloader)
+    pipeline = pot.create_pipeline(algorithms, engine)
```

Comment:
@eaidova What do you mean by opportunity? Do you need some API like callback to get opportunity collect progress?

Reply:
@alexsu52 I just mean provide a flag `print_progress` to turn on it  when we call `ptq.quantization` , if I right remember pot IEEngine is able to log it:
https://github.com/openvinotoolkit/openvino/blob/master/tools/pot/openvino/tools/pot/engines/ie_engine.py#L71
or maybe allow to configure preferable log level

callback api maybe useful for advanced users, but too complex for understanding, so just print any logs instead of 10h of siliency in my case will be pretty well :)

Reply:
What do you think about control it via logger and setting log level. I mean using nncf logger to control it. It wiil allow us to avoid additional input argument in the quantize method.

cc @KodiaqQ @l-bat @andrey-churkin @AlexKoff88

Reply:
could you please provide some code sample how it will looks?

Reply:
> could you please provide some code sample how it will looks?

If we are talking about the output of information about the progress of quantization than the code sample will not changed. 

If you want to disable logging than you can do the following:
```
from nncf.common.uteils.logging import disable_logging
disable_logging()
```

If you want to change the log level, for example, you need debug logs:
```
import logging
from nncf.common.uteils.logging import set_log_level
set_log_level(logging.DEBUG)
```

Reply:
looks like now if user not set log level, it is disabled by default, right? or some changes requires to enable it?

Reply:
If an user does not set log level, it is the INFO level enabled by default. It's mean that we print information about the progress of the quantization. The user can disable logger or change log level to get more information.

Reply:
I think that @eaidova means forwarding the logging level from NNCF to POT at the same level. I mean, if the NNCF has log-level INFO, it should be the same in the POT. The same situation for DEBUG log level and etc.  The fact is that POT regulates AC logging in the same manner inside using these levels.
@eaidova am I correct?

Reply:
Done